iothub/device/src/Transport/AmqpIoT/AmqpIoTConstants.cs

```diff
@@ -39,6 +40,29 @@ internal static class AmqpIoTConstants
         internal const string MethodName = "IoThub-methodname";
         internal const string Status = "IoThub-status";
 
+        // The tracking Id identifier, for an Amqp Error returned by the service.
+        public static readonly AmqpSymbol TrackingId = Vendor + ":tracking-id";
```

Comment:
how will this id be used? Is this introduced as a new field now?

Reply:
No, this has simply been moved over from `AmqpIoTErrorAdapter.TrackingId ` to `AmqpIoTConstants.TrackingId`.